.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: ruby
+sudo: false
```

Comment:
Could you also add below this line:

```yaml
cache:
  bundler: true
  bundler_args: --jobs=3 --retry=3 --without development
```

Read more about it at [here](http://docs.travis-ci.com/user/caching/).